services/invitations/src/simcore_service_invitations/web_server.py

```diff
@@ -0,0 +1,15 @@
+from typing import Literal
+
+import uvicorn
+
+
+def start(
+    log_level: Literal["info", "debug", "warning", "error"], reload: bool = False
+):
+    uvicorn.run(
+        "simcore_service_invitations.web_main:the_app",
+        host="0.0.0.0",  # nosec
+        port=8000,
+        log_level=log_level,
+        reload=reload,
+    )
```

Comment:
this has the advantage that we can start the service using the app's CLI. Uvicorn latest versions allows programatically calling the webserver and passing the fastapi app. 

The functionality is indeed duplicated, nonetheless the Dockerfile uses the "non-embedded" version to keep the same pattern as the rest of the services

I just wanted to give it a try and eventually suggest changing to this method since the latest versions of uvicorn already allows it.